### ISSUE TITLE
[Feature Request] Change Page With Button Press

### ISSUE THREAD

**Issue by TheRhysWyrill** [cy33hc/vita-launcher#11]
Would it be possible to change the page using a button press like L2/R2 if using controller (For PSTV users like myself etc) or the Left/Right side of the back touchpad on the Vita?

**Comment by cy33hc** [comment]
You can already change pages with L1/R1. Works for both PSTV and Vita.

**Comment by TheRhysWyrill** [comment]
That switches through the different tabs (Vita, PSP etc), what I meant is switching through the pages on say my Vita games with the press of a button instead of having to move the cursor to the edge of the displayed games to go up/down a page respectively.

**Comment by cy33hc** [comment]
Oh. I get it.

**Comment by TheRhysWyrill** [comment]
Yeah, sorry if I didn't word it all too well 😅 Would this be a possibility? Huge thanks for this launcher btw, being able to launch PSP games directly is so convenient!

**Comment by cy33hc** [comment]
Yeah. That's possible.

**Comment by cy33hc** [comment]
@TheRhysWyrill This is now partially implemented in Release 2.5 for using L2/R2 on PSTV. The back touch will come later, since it's a little harder to implement.

**Comment by rafaelferreira01** [comment]
@cy33hc Maybe use the right analog to change pages?
